lang/lang-es.js

```diff
@@ -0,0 +1,560 @@
+(function (window) {
+/**
+ * {virtualclass1} {virtualclass2} are elements you passes with getString function eg:-
+ *   virtualclass.lang.getString('operaBrowserIssue', ['opeara', 27]);
+ *   opera and 27 will be replaced over the {virtualclass1} and {virtualclass2} resepectively for particular line of language file.
+ * @type type
+ */
+const message = {
+    ActiveAll: 'Activar todo',
+    addcontext: 'Añadir marcador',
+    Addoption: 'Añadir opción',
+    addnew: 'Añadir nuevo',
+    adminusr: 'Administrador',
+    appPrerequites: 'Validar requisitos del sistema',
+    assign: 'Asignar',
+    assignDisable: 'Recuperar controles',
+    askplayMessage: '<span id="askplaymsg"> ¿Deberíamos empezar la sesión?</span><br /><span id="remaingmsg">Los datos restantes se descargarán en segundo plano.</span>',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 170. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)